ports/windows/alloc.c

```diff
@@ -0,0 +1,101 @@
+/*
```

Comment:
This looks like a copy of unix/alloc.c with like 3 lines changed. In such case I think it's better to modify unix/alloc.c instead to make it work with windows builds.